tests/test_index.py

```diff
@@ -0,0 +1,79 @@
+from typing import List
+
+import pytest
+
+import greenplumpython as gp
+from tests import db
+
+
+def test_op_on_consts(db: gp.Database):
+    regex_match = gp.operator("~")
```

Comment:
can we predefine those operators? Those are standard PG operators I think?

Reply:
Oh, right, the "standard python operators" are handled in another way.

> Which ones do you think we should predefine

No idea, probably some common ones, like regex matching. But it is up to you want to add those in the coming release. I am fine with both.